README.md

```diff
@@ -16,66 +16,69 @@ This repository is for Browser.html (the front-end). Active development of Graph
 We welcome contributions from anyone. See [CONTRIBUTING.md](https://github.com/browserhtml/browserhtml/blob/master/CONTRIBUTING.md) for help getting started.
 
 
-### Building and Running
+## Development
 
-The Browser.html UI is bundled with Servo. To run it, you'll need to build Servo.
+There are two major components to the browser.html application.
 
-First, [install Servo's prerequisites](https://github.com/servo/servo#setting-up-your-environment). Then, clone and build Servo:
+1. Local server that serves application UI.
+2. Client that is a application shell that connects to the server and renders served UI.
 
-``` sh
-git clone https://github.com/servo/servo
-cd servo
-./mach build -r
-```
-
-Finally, run Servo with the `--browserhtml` flag.
 
-``` sh
-./mach run -r --browserhtml
-```
+### Server
 
-
-### Developing the Front-end
-
-If you're working on the Browser.html front-end, you'll want to run the web app locally.
-
-**Prerequisites**: You'll need [Node](https://nodejs.org/) and NPM. Next, clone Browser.html and install its Node dependencies.
+If you're working on the Browser.html front-end, you'll need a [node][] to install all dependcies and run development toolchain.
```

Comment:
Nit 1: s/you'll need a [node][]/you'll need [Node.js][]/ (and the footnote fixed up.)
Nit 2: s/dependcies/dependencies/
Nit 3: s/run development/run the development/